test/Conversion/TorchToTcp/misc.mlir

```diff
@@ -30,3 +30,303 @@ func.func @torch.vtensor.literal() -> !torch.vtensor<[4],ui8> {
   %0 = torch.vtensor.literal(dense<[1, 2, 3, 4]> : tensor<4xui8>) : !torch.vtensor<[4],ui8>
   return %0 : !torch.vtensor<[4],ui8>
 }
+
+// -----
+
+// CHECK-LABEL:  @torch.aten.ones_f32()
+// CHECK-SAME:    -> !torch.vtensor<[2,3],f32> {
+// CHECK:         %[[V2:.*]] = torch.constant.int 2
+// CHECK:         %[[V3:.*]] = torch.constant.int 3
+// CHECK:         %[[T1:.*]] = tcp.const {value = dense<1.000000e+00> : tensor<f32>} : tensor<f32>
+// CHECK:         %[[T2:.*]] = torch_c.to_i64 %[[V2]]
+// CHECK:         %[[T3:.*]] = arith.index_cast %[[T2]] : i64 to index
+// CHECK:         %[[T4:.*]] = torch_c.to_i64 %[[V3]]
+// CHECK:         %[[T5:.*]] = arith.index_cast %[[T4]] : i64 to index
+// CHECK:         %[[T6:.*]] = tensor.expand_shape %[[T1]] [] : tensor<f32> into tensor<1x1xf32>
+// CHECK:         %[[T7:.*]] = tcp.broadcast  %[[T6]], %[[T3]], %[[T5]] {axes = [0, 1]} : tensor<1x1xf32>, index, index -> tensor<2x3xf32>
+// CHECK:         %[[T8:.*]] = torch_c.from_builtin_tensor %[[T7]] : tensor<2x3xf32> -> !torch.vtensor<[2,3],f32>
+// CHECK:         return %[[T8]] : !torch.vtensor<[2,3],f32>
+func.func @torch.aten.ones_f32() -> !torch.vtensor<[2,3],f32> {
+  %false = torch.constant.bool false
+  %none = torch.constant.none
+  %int2 = torch.constant.int 2
+  %int3 = torch.constant.int 3
+  %0 = torch.prim.ListConstruct %int2, %int3 : (!torch.int, !torch.int) -> !torch.list<int>
+  %cpu = torch.constant.device "cpu"
+  %1 = torch.aten.ones %0, %none, %none, %cpu, %false : !torch.list<int>, !torch.none, !torch.none, !torch.Device, !torch.bool -> !torch.vtensor<[2,3],f32>
```

Comment:
Can we have these tests with dynamic shapes instead?

You could have 1 static shape test, but otherwise we should have all dynamic shape tests.

In this case, you could take `int`s as inputs to the function, and use them in `ListConstruct`. Then the shape will be dynamic.